src/chains/ethereum/utils/src/types/fee-history.ts

```diff
@@ -0,0 +1,8 @@
+import { Data, Quantity } from "@ganache/utils";
+
+export type FeeHistory = {
+  oldestBlock: Data;
```

Comment:
This is the block number, which AFAIK should be Quantity. Check out [runtime-block.ts](https://github.com/trufflesuite/ganache/blob/develop/src/chains/ethereum/block/src/runtime-block.ts#L42) as a reference.

Reply:
:+1: